lib/Doctrine/ORM/Tools/SchemaValidator.php

```diff
@@ -261,11 +261,21 @@ public function validateClass(ClassMetadataInfo $class)
      * @return bool
      */
     public function schemaInSyncWithMetadata()
+    {
+        return count($this->getUpdateSchemaList()) === 0;
+    }
+
+    /**
+     * Returns the list of missing Database Schema updates.
+     *
+     * @return string[]
```

Comment:
You will probably be interested in the outcome of https://github.com/doctrine/orm/issues/9025

Reply:
`array<string>` then?